src/resourcetiming-decompression.js

```diff
@@ -197,18 +197,22 @@
     // Dimension data special type
     var SPECIAL_DATA_SCRIPT_TYPE = "2";
 
+    // Dimension data special type
+    var SPECIAL_DATA_SERVERTIMING_TYPE = "3";
+
     // Regular Expression to parse a URL
     var HOSTNAME_REGEX = /^(https?:\/\/)([^\/]+)(.*)/;
 
     /**
      * Decompresses a compressed ResourceTiming trie
      *
      * @param {object} rt ResourceTiming trie
+     * @param {array} st server timing entries lookup
      * @param {string} prefix URL prefix for the current node
      *
      * @returns {ResourceTiming[]} ResourceTiming array
      */
-    ResourceTimingDecompression.decompressResources = function(rt, prefix) {
+    ResourceTimingDecompression.decompressResources = function(rt, st, prefix) {
```

Comment:
I _believe_ that `prefix` is only passed in from the recursive call, but it's up to you.